src/components/TaskSummary/index.js

```diff
@@ -45,7 +53,7 @@ export class TaskSummary extends React.Component {
 
     onLoaded = () => this.setState({ loading: false })
 
-    onLoading = () => this.setState({ loading: true })
+    onLoading = () => this.setState({ loading: true, error: [] })
```

Comment:
Are you using `error` from the state or should this be removed now that you set the `errors` array in `initialState`?

Reply:
This should unset the errors when loading starts again, because you won't care about the errors from the previous upload, right?

Reply:
Sure, but then it should be `errors` (plural) and not `error` (singular).

Reply:
Good catch, this time there's not `error` left (except for the event, which is correct)